index.html

```diff
@@ -25,7 +26,174 @@ <h1>If These Lands Could Talk</h1>
 
 		<!-- Footer -->
 
+		<footer>
+			<div class="wrapper">
+				<div class="footer-main">
+					<div class="footer-logo-btn">
+						<!-- logo -->
+						<a href="./index.html">
+							<img
+								src="./assets/icons/global/itlct-logo.svg"
+								alt="If These Lands Could Talk Logo"
+							/>
+						</a>
+
+						<!-- Button for modal -->
+						<button class="btn-la">Land Acknowledgement</button>
+					</div>
+					<!-- Links list -->
+					<!-- need to replace with the real page links -->
+					<ul>
+						<li>
+							<a href="./index.html">Home</a>
+						</li>
+						<li>
+							<a href="#">Our Impact</a>
```

Comment:
Let's add the actual links here so that others can copy and paste this without having to update the links later. Talk to the other teams if you need to know what they are naming the html files.